src/__tests__/FixedSizeGrid.js

```diff
@@ -289,12 +289,17 @@ describe('FixedSizeGrid', () => {
         spyOn(console, 'warn');
 
         const renderer = ReactTestRenderer.create(
-          <FixedSizeGrid {...defaultProps} overscanCount={1} />
+          <FixedSizeGrid
+            {...defaultProps}
+            overscanCount={1}
+            overscanRowsCount={1}
+            overscanColumnsCount={1}
+          />
         );
         expect(console.warn).toHaveBeenCalledTimes(1);
         expect(console.warn).toHaveBeenLastCalledWith(
-          'The overscanCount prop has been deprecated. ' +
-            'Please use the overscanColumnsCount and overscanRowsCount props instead.'
+          'The overscanCount, overscanColumnsCount and overscanRowsCount props have been deprecated. ' +
```

Comment:
I think this message may be a bit confusing for people who were unaware of the old `overscanCount` prop. Seems like it would be better to split these into two warnings?

Reply:
OK, will do